src/components/sandboxLink.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import usePlatform from "./hooks/usePlatform";
+
+
+export enum Scenario {
+    Performance = 'performance',
+    Releases = 'releases',
+    Alerts = 'alerts',
+    Discover = 'discover',
+    Dashboards = 'dashboards',
+    Projects = 'projects',
+    OneDiscoverQuery = 'oneDiscoverQuery',
+    OneIssue = 'oneIssue',
+    OneBreadcrumb = 'oneBreadcrumb',
+    OneStackTrace = 'oneStackTrace',
+    OneTransaction = 'oneTransaction',
+    OneWebVitals = 'oneWebVitals',
+    OneTransactionSummary = 'oneTransactionSummary',
+    OneRelease = 'oneRelease',
+}
+
+type Props = {
+    children: React.ReactNode;
+    scenario?: Scenario;
+    projectSlug?: string;
+    errorType?: string;
+    platform?: string;
+}
+
+/**
+ * Obtains the URL to the sandbox start endpoint.
+ * @param param0.scenario: One of the scenarios. Determins where in the sandbox 
+ * the user will be landed.
+ * @param param0.projectSlug:
+ * One of react, python, react-native, android, iOS
+ * @param param0.errorType: A string matching the title of the error.
+ * @returns URL to the sandbox start endpoint
+ */
+export function getSandboxURL({ scenario, projectSlug, errorType }: {
+  scenario?: Scenario,
+  projectSlug?: string,
+  errorType?: string,
+} = {}) {
+  const url = new URL('https://try.sentry-demo.com/demo/start/');
+
+  if (scenario) url.searchParams.append('scenario', scenario);
+
+  if (projectSlug) url.searchParams.append('projectSlug', projectSlug);
+
+  if (errorType) url.searchParams.append('errorType', errorType);
+  url.searchParams.append('source', 'docs');
+
+  return url;
+};
+
+export const isSandboxHidden = () => process.env.GATSBY_HIDE_SANDBOX === '1';
+
+const SANDBOX_PLATFORMS: string[] = [
```

Comment:
@Neo-Zhixing I would probably type this as:
```
const SANDBOX_PLATFORMS = [
    'react',
    'python',
    'react-native',
    'android',
    'ios',
] as const;
```
Not sure why you want to explicitly type it as `string[]`

Reply:
Because then I have to cast it on line 77 but I guess it's fine